CMakeLists.txt

```diff
@@ -85,3 +103,19 @@ configure_file(src/strife-res.rc.in src/strife-res.rc)
 foreach(SUBDIR textscreen opl pcsound src)
     add_subdirectory("${SUBDIR}")
 endforeach()
+
+# Generate distribution packages with CPack.
+if(VCPKG_DIR)
+    set(CPACK_GENERATOR ZIP)
+    set(CPACK_STRIP_FILES TRUE)
+    set(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
+    if(CMAKE_SIZEOF_VOID_P STREQUAL 8)
+        set(SYS_POSTFIX "win64")
+    else()
+        set(SYS_POSTFIX "win32")
+    endif()
+    set(CPACK_ARCHIVE_DOOM_FILE_NAME "${PROGRAM_PREFIX}doom-${PROJECT_VERSION}-${SYS_POSTFIX}")
+    set(CPACK_ARCHIVE_HERETIC_FILE_NAME "${PROGRAM_PREFIX}heretic-${PROJECT_VERSION}-${SYS_POSTFIX}")
+    set(CPACK_ARCHIVE_HEXEN_FILE_NAME "${PROGRAM_PREFIX}hexen-${PROJECT_VERSION}-${SYS_POSTFIX}")
```

Comment:
There's also Chocolate Strife. 😉

Reply:
Yes, this was copied from Crispy 😄
There is also a `man` directory and CPack can generate `deb` packages. At this point, I decided to stop and listen to the opinion of the maintainers.